.gitignore

```diff
@@ -8,7 +8,7 @@
 /vscode-json
 /vscode-debugadapter
 /vscode-debugprotocol
-/superbol-free
+/superbol-free*
```

Comment:
You likely want to edit the corresponding field in `drom.toml` instead.

Reply:
(I did add a message to inform not to edit this file directly, I somebody probably removed it by mistake)